backend/tracim_backend/tests/functional/test_user.py

```diff
@@ -5034,7 +5099,7 @@ def test_api__set_user_info__err_403__user_itself(self):
         assert res['user_id'] == user_id
         assert res['profile'] == 'users'
 
-    def test_api__set_user_email__err_403__other_normal_user(self):
+    def test_api__set_user_profile__err_403__other_normal_user(self):
```

Comment:
`s/403/400/`

Reply:
This one is a right test, 403 is correct here. I will add comment.

Reply:
you right